src/python/pants/init/engine_initializer.py

```diff
@@ -278,7 +270,7 @@ def build_configuration_singleton() -> BuildConfiguration:
 
         @rule
         def symbol_table_singleton() -> SymbolTable:
-            return symbol_table
+            return symbol_table_from_registered_targets
```

Comment:
This feels "wrong" that the `SymbolTable` only includes registered targets and leaves off `objects` and CAFOs.